app/controllers/api/public/v1/application_controller.rb

```diff
@@ -2,11 +2,37 @@ module API
   module Public
     module V1
       class ApplicationController < ActionController::API
+        include Pagy::Backend
+
         rescue_from ActiveRecord::RecordNotFound, with: :jsonapi_404
 
         def jsonapi_404
           render jsonapi: nil, status: :not_found
         end
+
+      private
+
+        def paginate(scope)
```

Comment:
I noticed that these methods are being duplicated into all `application_controller.rb` files for the old api versions and the new public one. Is it worth just putting these methods into a `Paginatable` module and including that into those application controller classes?

Reply:
I'd rather opt for this duplication. I don't see the benefit of having this as shared code. The pagination code is fairly slim. For public/v1 we could probably drop the `paginate` method soon as i only added it for backwards compatibility as they is how the current code calls it

Reply:
yeah that's fair